src/applications/vaos/containers/DateTimeSelectPage.jsx

```diff
@@ -24,10 +24,17 @@ const pageTitle = 'Tell us the date and time you’d like your appointment';
 
 const missingDateError = 'Please select a preferred date for your appointment';
 
-export function getOptionsByDate(selectedDate, availableSlots = []) {
+export function getOptionsByDate(
+  selectedDate,
+  timezoneDescription,
+  availableSlots = [],
+) {
   return availableSlots.reduce((acc, slot) => {
     if (slot.start.split('T')[0] === selectedDate) {
-      const time = moment(slot.start);
+      let time = moment(slot.start);
+      if (slot.start.endsWith('Z')) {
```

Comment:
This is how I handled the slots coming from VSP in UTC and in facility time from VAR.

I'm hesitant to convert the facility times from VAR to UTC because I don't know if there is some weird timezone conversion logic in the backend that puts them in facility time in the first place. At least this way I'm not messing with the current slots and it's less likely to break for now.